express-delete/app.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.listen(3000, () => {
+  // console.log('Express server is listening on port 3000');
+});
```

Comment:
Although it works wherever you put `app.listen` it is traditionally put at the end of the file, as the last thing.